lxd/device/nic_bridged.go

```diff
@@ -246,13 +254,13 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 			} else {
 				// If MAAS isn't being used, then static IP cannot be used with unmanaged parent.
 				if d.config["ipv4.address"] != "" && d.config["maas.subnet.ipv4"] == "" {
-					return fmt.Errorf("Cannot use manually specified ipv4.address when using unmanaged parent bridge")
+					return fmt.Errorf("Cannot have ipv4.address as none unless using security.ipv4_filtering")
 				}
 			}
 
 			if shared.IsTrue(d.config["security.ipv6_filtering"]) {
 				if d.config["ipv6.address"] == "" {
-					return fmt.Errorf("IPv6 filtering requires a manually specified ipv6.address when using an unmanaged parent bridge")
+					return fmt.Errorf("Cannot have ipv6.address as none unless using security.ipv4_filtering")
```

Comment:
This shouldn't chance, as if IPv6 address is "none" the user is telling us to block all IPv6 traffic.